### ISSUE TITLE
Specify a port

### ISSUE THREAD

**Issue by Nicoraschia** [AsBuiltReport/AsBuiltReport.Fortinet.FortiGate#86]
### Description

First of all, thank you for this enormous contribution. Could you help me connect to the FGT on a port other than the default (port 440)?
I am  trying this:
New-AsBuiltReport -Report Fortinet.FortiGate -Target 192.168.0.100 -Username $user -Password $password  

### Before submitting

- [X] I have read [the documentation](https://www.asbuiltreport.com/user-guide/new-asbuiltconfig), and referred to the [known issues](https://www.asbuiltreport.com/user-guide/known-issues/) before submitting this change request.
- [X] I have checked for previously opened & closed [issues](https://github.com/AsBuiltReport/AsBuiltReport.Fortinet.FortiGate/issues) before submitting this change request.

**Comment by tpcarman** [comment]
Please refer to the [documentation](https://github.com/AsBuiltReport/AsBuiltReport.Fortinet.FortiGate?tab=readme-ov-file#options). 

You can either modify the default `AsBuiltReport.Fortinet.Fortigate.json` file or create a specific one by running `New-AsBuiltReportConfig -Report Fortinet.Fortigate -FolderPath <folder> -Filename <filename>`.

Then, when generating a report you can specify a specific report configuration JSON file using the `ReportConfigFilePath` parameter with `New-AsBuiltReport` command.

e.g. `New-AsBuiltReport -Report Fortinet.Fortigate -Target 192.168.0.100 -Username $user -Password $password -ReportConfigFilePath "C:\scripts\AsBuiltReport.Fortinet.Fortigate.json" `

**Comment by Nicoraschia** [comment]
Hi @tpcarman
The port worked ok, But the report is all blank, and I don't see any errors using -verbose

`
VERBOSE: Saving As Built Report configuration file 'AsBuiltReport.json' to path 'C:\Users\nicolasr\Desktop\reporte'.
VERBOSE: Config.Report.Author = nicolasr
VERBOSE: Config.UserFolder.Path =
VERBOSE: Config.Company.FullName =
VERBOSE: Config.Company.Contact =
VERBOSE: Config.Company.Phone =
VERBOSE: Config.Company.Email =
VERBOSE: Config.Company.ShortName =
VERBOSE: Config.Company.Address =
VERBOSE: Config.Email.Port =
VERBOSE: Config.Email.Credentials =
VERBOSE: Config.Email.Server =
VERBOSE: Config.Email.To =
VERBOSE: Config.Email.From =
VERBOSE: Config.Email.UseSSL =
VERBOSE: Config.Email.Body =
VERBOSE: Populating RepositorySourceLocation property for module AsBuiltReport.Core.
VERBOSE: Loading module from path 'C:\Program Files\WindowsPowerShell\Modules\AsBuiltReport.Core\1.4.0\AsBuiltReport.Core.psm1'.
VERBOSE: Populating RepositorySourceLocation property for module AsBuiltReport.Fortinet.FortiGate.
VERBOSE: Loading module from path 'C:\Program Files\WindowsPowerShell\Modules\AsBuiltReport.Fortinet.FortiGate\0.4.0\AsBuiltReport.Fortinet.FortiGate.psm1'.
VERBOSE: Loading AsBuiltReport.Fortinet.Fortigate report configuration file from path 'C:\Users\nicolasr\Desktop\reporte\Nico.json'.
VERBOSE: Setting report filename to 'Fortinet FortiGate As Built Report'.
VERBOSE: Populating RepositorySourceLocation property for module AsBuiltReport.Core.
VERBOSE: Loading module from path 'C:\Program Files\WindowsPowerShell\Modules\AsBuiltReport.Core\1.4.0\AsBuiltReport.Core.psm1'.
WARNING: [ 10:10:19:511 ] [  Module  ] - AsBuiltReport.Core 1.4.0 is currently installed.
VERBOSE: [ 10:10:35:248 ] [ Document ] - Document 'Fortinet FortiGate As Built Report' processing started.
VERBOSE: [ 10:10:35:310 ] [ Document ] - Please wait while the Fortinet Fortigate As Built Report is being generated.
VERBOSE: [ 10:10:35:310 ] [ Document ] - Executing report style script from path 'C:\Program
Files\WindowsPowerShell\Modules\AsBuiltReport.Core\1.4.0\AsBuiltReport.Core.Style.ps1'.
VERBOSE: [ 10:10:35:326 ] [ Document ] - Setting global document options.
VERBOSE: [ 10:10:35:326 ] [ Document ] - Enabling section/heading numbering.
VERBOSE: [ 10:10:35:326 ] [ Document ] - Setting default font(s) to 'Segoe Ui'.
VERBOSE: [ 10:10:35:326 ] [ Document ] - Setting page top margin to '25.05'mm.
VERBOSE: [ 10:10:35:326 ] [ Document ] - Setting page right margin to '25.05'mm.
VERBOSE: [ 10:10:35:326 ] [ Document ] - Setting page bottom margin to '25.05'mm.                                                                                       VERBOSE: [ 10:10:35:326 ] [ Document ] - Setting page left margin to '25.05'mm.                                                                                         VERBOSE: [ 10:10:35:342 ] [ Document ] - Setting page size to 'A4'.                                                                                                     VERBOSE: [ 10:10:35:342 ] [ Document ] - Setting page orientation to 'Portrait'.                                                                                        VERBOSE: [ 10:10:35:342 ] [ Document ] - Setting page height to '297'mm.
VERBOSE: [ 10:10:35:342 ] [ Document ] - Setting page width to '210'mm.
VERBOSE: [ 10:10:35:342 ] [ Document ] - Setting document style 'Title'.
VERBOSE: [ 10:10:35:357 ] [ Document ] - Setting document style 'Title2'.
VERBOSE: [ 10:10:35:357 ] [ Document ] - Setting document style 'Title3'.
VERBOSE: [ 10:10:35:357 ] [ Document ] - Setting document style 'Heading1'.
VERBOSE: [ 10:10:35:357 ] [ Document ] - Setting document style 'Heading2'.
VERBOSE: [ 10:10:35:373 ] [ Document ] - Setting document style 'Heading3'.
VERBOSE: [ 10:10:35:373 ] [ Document ] - Setting document style 'Heading4'.
VERBOSE: [ 10:10:35:373 ] [ Document ] - Setting document style 'NOTOCHeading4'.
VERBOSE: [ 10:10:35:388 ] [ Document ] - Setting document style 'Heading5'.
VERBOSE: [ 10:10:35:388 ] [ Document ] - Setting document style 'NOTOCHeading5'.
VERBOSE: [ 10:10:35:388 ] [ Document ] - Setting document style 'Heading6'.
VERBOSE: [ 10:10:35:404 ] [ Document ] - Setting document style 'NOTOCHeading6'.
VERBOSE: [ 10:10:35:404 ] [ Document ] - Setting document style 'NOTOCHeading7'.
VERBOSE: [ 10:10:35:404 ] [ Document ] - Setting document style 'Normal'.
VERBOSE: [ 10:10:35:404 ] [ Document ] - Setting document style 'Header'.
VERBOSE: [ 10:10:35:420 ] [ Document ] - Setting document style 'Footer'.
VERBOSE: [ 10:10:35:420 ] [ Document ] - Setting document style 'TOC'.
VERBOSE: [ 10:10:35:420 ] [ Document ] - Setting document style 'TableDefaultHeading'.
VERBOSE: [ 10:10:35:420 ] [ Document ] - Setting document style 'TableDefaultRow'.
VERBOSE: [ 10:10:35:435 ] [ Document ] - Setting document style 'Critical'.
VERBOSE: [ 10:10:35:435 ] [ Document ] - Setting document style 'Warning'.
VERBOSE: [ 10:10:35:435 ] [ Document ] - Setting document style 'Info'.
VERBOSE: [ 10:10:35:451 ] [ Document ] - Setting document style 'OK'.
VERBOSE: [ 10:10:35:451 ] [ Document ] - Setting document style 'Caption'.
VERBOSE: [ 10:10:35:467 ] [ Document ] - Setting table style 'TableDefault'.
VERBOSE: [ 10:10:35:467 ] [ Document ] - Setting table style 'Borderless'.
VERBOSE: [ 10:10:35:467 ] [ Document ] - Processing document header started.
VERBOSE: [ 10:10:35:467 ] [ Document ] - Processing paragraph 'Fortinet FortiGate As Built Report -[..]'.
VERBOSE: [ 10:10:35:482 ] [ Document ] - Processing document header completed.
VERBOSE: [ 10:10:35:482 ] [ Document ] - Processing document footer started.
VERBOSE: [ 10:10:35:498 ] [ Document ] - Processing paragraph 'Page <!# PageNumber #!>'.
VERBOSE: [ 10:10:35:498 ] [ Document ] - Processing document footer completed.
VERBOSE: [ 10:10:35:513 ] [ Document ] - Processing blank line.
VERBOSE: [ 10:10:35:513 ] [ Document ] - Processing image 'AsBuiltReport Logo'.
VERBOSE: [ 10:10:35:513 ] [ Document ] - Processing blank line.
VERBOSE: [ 10:10:35:529 ] [ Document ] - Processing paragraph 'Fortinet FortiGate As Built Report'.
VERBOSE: [ 10:10:35:529 ] [ Document ] - Processing blank line.
VERBOSE: [ 10:10:35:529 ] [ Document ] - Processing table 'Cover Page'.
VERBOSE: [ 10:10:35:529 ] [ Document ] - Processing page break.
VERBOSE: [ 10:10:35:529 ] [ Document ] - Processing table of contents 'Table of Contents'.
VERBOSE: [ 10:10:35:545 ] [ Document ] - Processing page break.
VERBOSE: [ 10:10:35:685 ] [ Document ] - Connect to 192.168.0.100 : FG100F (FG100FTK2xxxxx)
VERBOSE: [ 10:10:35:764 ] [ Document ] - Processing section 'FGT100F Configuration' started.
VERBOSE: [ 10:10:35:779 ] [ Document ] - Processing paragraph 'The following provides as-built docu[..]'.
VERBOSE: [ 10:10:35:779 ] [ Document ] - Processing blank line.
WARNING: [ 10:10:35:873 ] [ Document ] - The term 'Get-AbrFgtFortiCare' is not recognized as the name of a cmdlet, function, script file, or operable program. Check
the spelling of the name, or if a path was included, verify that the path is correct and try again.
VERBOSE: [ 10:10:35:904 ] [ Document ] - Document 'Fortinet FortiGate As Built Report' processing completed.
VERBOSE: [ 10:10:35:904 ] [ Document ] - Total processing time '0.65' seconds.
Fortinet Fortigate As Built Report 'Fortinet FortiGate As Built Report' has been saved to 'C:\User\reporte\'.
`
![image](https://github.com/user-attachments/assets/cf6f62c0-c959-4c3a-ba80-eb6898ae7797)

**Comment by Nicoraschia** [comment]
+ @msoukhomlinov

**Comment by alagoutte** [comment]
there is the following error :
`the term 'Get-AbrFgtFortiCare' is not recognized as the name of a cmdlet, function, script file, or operable program. Check`

can you (re)try

 ```powershell
import-Module asBuiltReport.Fortinet.FortiGate`

and if always don't work what do you have

 ```powershell
Get-Command -Module asBuiltReport.Fortinet.FortiGate`

also what your powershell release ?

**Comment by Nicoraschia** [comment]
```
PS C:\Users\User01\AsBuiltReport> Import-Module AsBuiltReport.Fortinet.FortiGate
WARNING: The names of some imported commands from the module 'PowerFGT' include unapproved verbs that might make them
less discoverable. To find the commands with unapproved verbs, run the Import-Module command again with the Verbose
parameter. For a list of approved verbs, type Get-Verb.
C:\Program
Files\WindowsPowerShell\Modules\AsBuiltReport.Fortinet.FortiGate\0.4.0\AsBuiltReport.Fortinet.FortiGate.psm1 : Failed
to import function C:\Program
Files\WindowsPowerShell\Modules\AsBuiltReport.Fortinet.FortiGate\0.4.0\Src\Private\Get-AbrFgtFortiCare.ps1: At
C:\Program
Files\WindowsPowerShell\Modules\AsBuiltReport.Fortinet.FortiGate\0.4.0\Src\Private\Get-AbrFgtFortiCare.ps1:177 char:69
+ ...             $licenseStatus = $license.status -eq 'licensed' ? 'Licens ...
+                                                                 ~
Unexpected token '?' in expression or statement.
At line:1 char:1
+ Import-Module AsBuiltReport.Fortinet.FortiGate
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,AsBuiltReport.Fortinet.FortiGate.psm1
```
```
Name                           Value
----                           -----
PSVersion                      5.1.20348.2582
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.20348.2582
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

```
`PS C:\Users\nicolasr\AsBuiltReport> Import-Module AsBuiltReport.Fortinet.FortiGate -Verbose
VERBOSE: Loading module from path 'C:\Program
Files\WindowsPowerShell\Modules\AsBuiltReport.Fortinet.FortiGate\0.4.0\AsBuiltReport.Fortinet.FortiGate.psd1'.
VERBOSE: Importing function 'Invoke-AsBuiltReport.Fortinet.FortiGate'.`

**Comment by alagoutte** [comment]
Ok..

**Comment by alagoutte** [comment]
Fixed witht #87 and with 0.4.1
